icees_api/handlers.py

```diff
@@ -126,10 +130,12 @@ def edit_cohort(
     """Cohort discovery - create a cohort or find an existing one.
```

Comment:
@karafecho Noticed this line in the comment for edit_cohort is not accurate, which appears to be a copy and paste error from cohort discovery endpoint.